Bottom: 4fe8ae6c5f8d69975ca02fd6c2bc97f5ce4f29e4
Top:    0c16d96799f01f4c46ae319379157c2411da6573
Author: Peter Clifton <peter@clifton-electronics.co.uk>
Date:   2016-12-04 19:24:29 +0000

Disable small edge EPSILON hack for computing edge normals


---

diff --git a/src/hid/step/object3d_step.c b/src/hid/step/object3d_step.c
index a04d8eb..468d966 100644
--- a/src/hid/step/object3d_step.c
+++ b/src/hid/step/object3d_step.c
@@ -227,7 +227,7 @@ object3d_to_step_body_fragment (step_file *step,
           dir_y = dv->y - ov->y;
           dir_z = dv->z - ov->z;
 
-#if 1
+#if 0
           /* XXX: This avoids the test file step_outline_test.pcb failing to display properly in freecad when coordinates are slightly rounded */
           if (dir_x < EPSILON && -dir_x < EPSILON &&
               dir_y < EPSILON && -dir_y < EPSILON &&
